tests/tga/bug00247.c

```diff
@@ -0,0 +1,14 @@
+#include <stdio.h>
+
+#include "gd.h"
+#include "gdtest.h"
+
+int main(int argc, char **argv)
+{
+    gdImagePtr im;
+    FILE *fp = gdTestFileOpen("tga/bug00247.tga");
+    im = gdImageCreateFromTga(fp);
+    if (im != NULL)
+	gdImageDestroy(im);
+    return 0;
+}
```

Comment:
This test doesn't seem to verify that `im` is supposed to be NULL. What about:

    im = gdImageCreateFromTga(fp);
    gdTestAssert(im == NULL);
    fclose(fp);
    return gdNumFailures();